js/wp-seo-metabox.js

```diff
@@ -496,18 +653,18 @@ jQuery(document).ready(function () {
 	var focuskwhelptriggered = false;
 	jQuery(document).on('change', '#' + wpseoMetaboxL10n.field_prefix + 'focuskw', function () {
 		var focuskwhelpElm = jQuery('#focuskwhelp');
-		if (jQuery('#' + wpseoMetaboxL10n.field_prefix + 'focuskw').val().search(',') != -1) {
+		if (jQuery('#' + wpseoMetaboxL10n.field_prefix + 'focuskw').val().search(',') !== -1) {
 			focuskwhelpElm.click();
 			focuskwhelptriggered = true;
 		} else if (focuskwhelptriggered) {
-			focuskwhelpElm.qtip("hide");
+			focuskwhelpElm.qtip('hide');
 			focuskwhelptriggered = false;
```

Comment:
this line is unneccesary since <code> focuskwhelptriggered</code> is already set on line 653